protos/odpf/stencil/stencil.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
```

Comment:
Any reason we are checking in this file here? I thought we have proton repo for this?

Reply:
Will move this proto to proton.